store/postgres/src/relational_queries.rs

```diff
@@ -2617,6 +2659,108 @@ impl<'a> SortKey<'a> {
             }
         }
 
+        fn with_child_object_key<'a>(
+            parent_table: &'a Table,
+            child_table: &'a Table,
+            join_attribute: String,
+            derived: bool,
+            attribute: String,
+            direction: &'static str,
+        ) -> Result<SortKey<'a>, QueryExecutionError> {
+            let sort_by_column = child_table.column_for_field(&attribute)?;
+            if sort_by_column.is_fulltext() {
+                Err(QueryExecutionError::NotSupported(
+                    "Sorting by fulltext fields".to_string(),
+                ))
+            } else if sort_by_column.is_primary_key() {
+                Err(QueryExecutionError::NotSupported(
+                    "Sorting by id".to_string(),
+                ))
```

Comment:
Why is this not allowed? The example for the PR mentions sorting by `capital__id`

Reply:
I fixed it, only sorting interfaces by id is not supported so far